app/models/language.rb

```diff
@@ -2,44 +2,12 @@
 #
 # Table name: languages
 #
+#  created_at :datetime
 #  id         :integer          not null, primary key
+#  locale     :string
 #  name       :string
-#  code       :string
-#  created_at :datetime
 #  updated_at :datetime
 #
 
 class Language < ActiveRecord::Base
```

Comment:
:D

On Wed, Feb 3, 2016 at 12:20 PM, Melody <notifications@github.com> wrote:

> In app/models/language.rb
> <https://github.com/sassafrastech/madeline_system/pull/43#discussion_r51754545>
> :
>
> >  #  name       :string
> > -#  code       :string
> > -#  created_at :datetime
> >  #  updated_at :datetime
> >  #
> >
> >  class Language < ActiveRecord::Base
>
> That's a good option, I'd considered it but wasn't sure how well it would
> actually work or whether there was a possibility there was other metadata
> we'd want/need to store about a language, so I'd decided to leave the
> model. Can switch that over though and delete the model entirely. Always
> happy to detonate a little more code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sassafrastech/madeline_system/pull/43/files#r51754545>
> .
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech

Resident, Touchstone Cohousing
touchstonecohousing.org